index.html

```diff
@@ -39,6 +39,156 @@ <h1>I'm José</br>
             </li>
         </ul>
     </section>
+    <section class="wrapper">
+         <article>
+            <img src="images/Snapshoot Portfolio.svg">
+            <div class="leftblock">
+                <h2>Tonic</h2>
+                <div class="ref">
+                    <span class="client">Canopy</span>
+                    <span><i class="fa fa-circle fa-xs"></i></span>
+                    <span>Back End Dev</span>
+                    <span><i class="fa fa-circle"></i></span>
+                    <span class="year">2015</span>
+                </div>
+                <p class="primarytext">A daily selection of privately personalized reads; no accounts or sign-ups required.</p> 
+            </div>
+            <ul class="tags">
+                <li class="tag">html</button>
+                <li class="tag">css</button>
+                <li class="tag">javascript</button>
+            </ul>
+            <div class="ref">
+            <button>See project</button>
```

Comment:
Good job so far, but as the previous reviewer mentioned it's important to add the button behaviors, so please try to implement the hover, pressed, and disabled behaviors for the buttons.


![image](https://user-images.githubusercontent.com/34779928/192640262-48e223d1-e9a1-40dc-9127-d76a3cb6312a.png)